lib/menu.ts

```diff
@@ -6,7 +6,7 @@
  * entirely by a custom item. For an item to be a selectable menu item, it needs `tabindex=-1`
  * attribute set. If unset, or if the "disabled" class is set, the item will not be selectable.
  *
- * Further, if `dom.data('menuItemSelected', (yesNo: boolean, elem) => {})` is set, that callback
+ * Further, if `dom.data('menuItemSelected', () => (yesNo: boolean, elem) => {})` is set, that callback
```

Comment:
I guess this would make it correct: `dom.dataElem(elem, 'menuItemSelected', (yesNo: boolean, elem) => {})`. But trying to set a function with `dom.data()` is indeed broken, and the workaround is confusing...